spec/scrub_output_structure_spec.rb

```diff
@@ -42,10 +42,9 @@
     expect { test.date_subdir!("log", "JAN 1ST LAST YEAR") }.to raise_error(StandardError)
   end
 
-  # last
-  it "removes itself" do
-    expect(Dir.exist?(test.member_dir)).to be(true)
-    expect(test.remove_recursive!).to be(true)
-    expect(Dir.exist?(test.member_dir)).to be(false)
+  it "otherwise an after-all but rubocop does not like" do
```

Comment:
What's rubocop complaining about? Disabling the cop would be preferable to working around it like this, IMO